cmd/juju/cloud/add.go

```diff
@@ -298,8 +298,8 @@ func (c *AddCloudCommand) Run(ctxt *cmd.Context) error {
 
 	if c.controllerName == "" {
 		if !c.Local {
-			ctxt.Stdout.Write(
-				[]byte("There are no controllers running.\nAdding cloud to local cache so you can use it to bootstrap a controller.\n"))
+			ctxt.Infof(
```

Comment:
I was happy with the initial message being written directly to stdout. I ummed and aahed when I first reviewed as to whether or not this should be written directly or using the Infof function. In the end I settled on writing directly because this part of the message we don't want hidden if they use --quiet as it actually impacts behaviour.